index.html

```diff
@@ -173,7 +176,8 @@ <h1>Multi-Post<br>
             </div>  
         </div>
     </section>
-    <section class="about-me">
+    <section class="about-me" id="about-me">
+     <div class="card-top">
      <h1>About<br>Myself</h1>
```

Comment:
- [ ] Kindly consider using the same typos(font), and color as the template in Figma.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201391749-5c52a3c6-b538-4d81-8609-47ee56506c61.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201391836-727cdaa4-ad4b-49e8-9bdd-524e60789302.png)